### PR TITLE
refactor: convert employee to array

### PR DESCRIPTION
**Pull request by Xillians**
This is a redesign of the employees page, with this design idea:

Rather than creating 1 component and filling out the data, we add it to a list and use a foreach on the list. on one hand, this list creates technically longer code, but in a cognitive perspective, all we see here is a list and set the employee accordingly. List follows pattern as placed, hence the comment names for ease of navigation. 

The main benefit here is consistency and that all we need to do for further recruitment is to add something to an object 💙

### CODE REVIEW

File: src/components/employees.svelte [hunk 1]
```diff
@@ -9,139 +9,175 @@
...
+      email: 'simen@bjerk.io',
+      image: simen,
+      socialMedias: {
+        linkedin: {
+          url: 'https://www.linkedin.com/in/simenandre/'
+        },
+        github: {
+          url: 'https://github.com/simenandre'
+        }
+      }
```
Comment (braaar) [inline src/components/employees.svelte:46]:
```suggestion
      linkedin: 'https://www.linkedin.com/in/simenandre/',
      github: 'https://github.com/simenandre',
```

**Review by braaar — COMMENTED** [review]
I'm not sure if this is useful. This list adds a new data shape that is only used to fill in the individual props on the Employee component. If we want to iterate over a list of objects we should probably make an interface and refactor the component so that it takes in just the employee object, but I'm not sure if that is good practice for purely hard coded content like this. It also gives us less intuitive control over which employee is placed where in the markup, I'd say.

I suppose it's a step in the right direction if we want to add a CMS some day, though.

File: src/components/employees.svelte (hunk 1 above)
Comment (braaar) [reply src/components/employees.svelte:46]: I would either go for something simple like this or make this an array that can be passed directly to the employee component.

**Comment by simenandre** [comment]
I don't think we want a CMS, probably not for this. Closing

**Pull request closed without merging**